turbo.json

```diff
@@ -27,15 +27,6 @@
       ],
       "outputs": ["pnpm-lock.yaml"]
     },
-    "build:idl": {
```

Comment:
I moved this to the package-specific turbo.json because it's very unlikely any other packages will ever implement this